package.json

```diff
@@ -22,7 +23,10 @@
     "extends": [
       "react-app",
       "react-app/jest"
-    ]
+    ],
+    "rules": {
+      "jsx-quotes": ["error", "prefer-double"]
```

Comment:
ta da, a jsx only double-quotes rule

Reply:
Awesome start! However I would recommend install `eslint-plugin-react` and add 
```
 "eslint:recommended",
 "plugin:react/recommended"
```
as your default linting rules. This is a basic starting point for all companies that is using React so it's better to be familiar with the rules instead of having custom rules.